otter/test/test_cloudfeeds.py

```diff
@@ -0,0 +1,288 @@
+"""
+Tests for otter.cloudfeeds
+"""
+
+from functools import partial
+
+from effect import Effect, TypeDispatcher
+from effect.twisted import deferred_performer
+
+import mock
+
+from twisted.internet.defer import fail, succeed
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.constants import ServiceType
+from otter.http import TenantScope, has_code, service_request
+from otter.log.cloudfeeds import (
+    CloudFeedsObserver,
+    UnsuitableMessage,
+    add_event,
+    prepare_request,
+    request_format,
+    sanitize_event
+)
+from otter.test.utils import CheckFailure, mock_log
+from otter.util.retry import (
+    ShouldDelayAndRetry,
+    exponential_backoff_interval,
+    retry_times
+)
+
+
+def sample_event_pair():
+    """
+    Sample pair of event dict that observer gets and its corresponding
+    cloud feed event
+    """
+    return {
+        "scaling_group_id": "gid",
+        "policy_id": "pid",
+        "webhook_id": "wid",
+        "username": "abc",
+        "desired_capacity": 5,
+        "current_capacity": 3,
+        "message": ("human", ),
+        "time": 0
+    }, {
+        "scalingGroupId": "gid",
+        "policyId": "pid",
+        "webhookId": "wid",
+        "username": "abc",
+        "desiredCapacity": 5,
+        "currentCapacity": 3,
+        "message": "human"
+    }
+
+
+class SanitizeEventTests(SynchronousTestCase):
+    """
+    Tests for :func:`otter.cloudfeeds.sanitize_events`
+    """
+
+    def setUp(self):  # noqa
```

Comment:
You probably have a stale plugin still installed then. Try `pip list`, or recreate your virtualenv, or figure out what the package name is. (I think it's `pep8-naming`.)